### ISSUE TITLE
Symfony\Component\Debug\Exception\FatalThrowableError Call to undefined function MarcReichel\IGDBLaravel\starts_with()

### ISSUE THREAD

**Issue by dominik-wbz** [marcreichel/igdb-laravel#15]
Hello, 

after fresh install laravel 6, I get the following error:

Symfony\Component\Debug\Exception\FatalThrowableError Call to undefined function MarcReichel\IGDBLaravel\starts_with()

```
<?php

namespace App\Http\Controllers;

use MarcReichel\IGDBLaravel\Models\Game;

class TestController extends Controller {
	function test() {
		$game = Game::find(1905);

		if ($game) {
			echo $game->name; // Will output "Fortnite"
		}
	}
}
```

Can you please help?

**Comment by marcreichel** [comment]
I going to fix this tomorrow and will provide a new release

**Comment by dominik-wbz** [comment]
Thaaank you man :)

**Comment by marcreichel** [comment]
Fixed with Release v0.10.1
